templates/contact.html

```diff
@@ -6,6 +6,6 @@
 <h2>Contact</h2>
 <p>Hi, I'm pajlada! I'm the current maintainer and developer of {{ bot.name }}.</p>
 <p>{{ bot.name }} is running on the <strong>pajbot</strong> source, which is coded in Python.</p>
-<p>Got any questions or comments about the bot or the website? <a href="http://twitch.tv/message/compose?to=pajlada">Let me know by sending me a twitch message!</a></p>
+<p>Got any questions or comments about the bot or the website? <a href="https://discord.gg/qbRE8WR">You can join discord for support!</a> Or just message me: pajlada#2107</p>
```

Comment:
Uhh this kind of mixing-in of changes with other stuff in the PR is not exactly ideal

Reply:
PepeLaugh

Reply:
This is why it's wiser to do your work on a branch different then master, just as a tip for next time

Reply:
Yeah I found that out soon afterwards PepeLaugh